File: tracks/python/exercises/02-first-workflow/practice/workflows.py
from temporalio import workflow


@workflow.defn
class TaglineWorkflow:
    @workflow.run
    async def run(self, brand: str) -> str:
        workflow.logger.info("Generating tagline for %s", brand)
        # TODO: Part B — return the tagline: the brand name, a colon and a space,
        # then "Taste the Meow!"  e.g.  "CatNip Cola: Taste the Meow!"
        return "TODO"

File: tracks/typescript/exercises/02-first-workflow/practice/workflows.ts
import { log } from '@temporalio/workflow';

export async function taglineWorkflow(brand: string): Promise<string> {
  log.info('Generating tagline', { brand });
  // TODO: Part B — return the tagline: the brand name, a colon and a space,
  // then "Taste the Meow!"  e.g.  "CatNip Cola: Taste the Meow!"
  return 'TODO';
}
